src/components/correct-name/CorrectName.vue

```diff
@@ -68,9 +68,9 @@ import { Component, Emit, Prop } from 'vue-property-decorator'
 import { NameRequestIF } from '@bcrs-shared-components/interfaces'
 import { CorrectNameOptions } from '@bcrs-shared-components/enums'
 import { CorpTypeCd } from '@bcrs-shared-components/corp-type-module'
-import CorrectCompanyName from '@bcrs-shared-components/correct-name/CorrectCompanyName.vue'
-import CorrectNameToNumber from '@bcrs-shared-components/correct-name/CorrectNameToNumber.vue'
-import CorrectNameRequest from '@bcrs-shared-components/correct-name/CorrectNameRequest.vue'
+import CorrectCompanyName from './CorrectCompanyName.vue'
```

Comment:
Changed these to relative imports

Reply:
Good idea. Dunno what I was thinking when I did this.